### PR TITLE
46 - Added content-type to resources in server push.

### LINKED ISSUE #46: Push fails when using javascript modules
**Version**: v0.4.0
**Browser**: Chrome

**Problem**: 
When importing resources using javascript modules Chrome fails to receive and parse the server pushed resources because there is no `content-type` header for them.

```html
<script type="module" src="myJsFile.js"></script>
```

Message:
```Failed to load module script: The server responded with a non-JavaScript MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec.```

### PR DESCRIPTION
**Pull request by douglascvas**
Added `content-type` to server push. Fixes #46

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,18 +44,21 @@
...
     "@types/semver": "^5.5.0",
-    "ava": "^1.0.0",
+    "ava": "^1.3.1",
     "codecov": "^3.0.1",
     "get-port": "^4.0.0",
     "gts": "^0.9.0",
     "js-green-licenses": "^0.5.0",
-    "nyc": "^13.0.0",
+    "nyc": "^13.3.0",
+    "requirejs": "^2.3.6",
```
Comment (ofrobots) [inline package.json]: Is this needed?

**Comment by ofrobots** [comment]
You can use this image instead: 
image

File: package.json (hunk 1 above)
Comment (douglascvas) [reply package.json]: I have no idea how did requirejs got into this, unless it has been added by `npm audit fix --force`. I am removing it anyway

**Comment by douglascvas** [comment]
> You can use this image instead:
> image

Awesome! Thank you @ofrobots

**Pull request merged**